Some improvements

liubko opened the pull request:
- Update `ResponsiveContainer` / `LineChart` / `BarChart`
- add `PxOrPrc` module
- add examples
- format code with `refmt`

liubko commented:
@rusty-key let me know what you think. I'm planning to cover more component's API here or preferably in a new PR

liubko commented:
Just learned a trick from Javier need to use `_type` instead of `type_`.

Will update PR tomorrow

rusty-key commented on examples/src/LineChartExample.re:
```diff
@@ -0,0 +1,16 @@
+        <Legend align="left" iconType="circle" />
+      </LineChart>
+    </ResponsiveContainer>,
+};
```
Can you please add `"files.insertFinalNewline": true,` to your editor config?

rusty-key commented:
https://github.com/ahrefs/bs-recharts/pull/1/files#diff-09d6724bde495fd0c2ba310fd66df417

Let's stick with yarn in this package?

rusty-key commented:
Other than that it's awesome!
By the way, do we want to cover package-specific types in documentation?

rusty-key commented on src/Utils.re:
```diff
@@ -0,0 +1,37 @@
+module PxOrPrc = {
+  type t;
+  type arg =
+    | Px(int)
```
I am a little bit afraid of this generic names. What will happen if there will be another package which defines same types?

liubko commented:
- updated other components as well Line/Bar/CasterianGrid/Casterianaxis/Tooltip/Legend

> By the way, do we want to cover package-specific types in documentation?

- updated Readme

@rusty-key please take a look again

Eric-Arellano commented:
@rusty-key @liubko these bindings are great! Thank you for making these. They're saving me a lot of time on project.

(Wasn't sure where to put this)
